package.json

```
@@ -4,7 +4,8 @@
   "scripts": {
     "build": "webpack -p --progress --config webpack.config.js",
     "dev-build": "webpack --progress -d --config webpack.config.js",
-    "watch": "webpack --progress -d --config webpack.config.js --watch"
+    "watch": "webpack --progress -d --config webpack.config.js --watch",
```

I think `-d` is for `--debug`? I prefer long-options over flags (when available) for obviousness
```suggestion
    "watch": "webpack --progress --debug --config webpack.config.js --watch",
```